components/woodpecker_co/utils.mjs

```diff
@@ -0,0 +1,36 @@
+export const orderOptions = [
```

Comment:
Can you put this file inside a `common/` folder so it's not mistaken as a component?